tests/unit_and_functional/conftest.py

```diff
@@ -81,6 +93,26 @@ def k8s_env_vars() -> Iterable[bool]:
     del os.environ['KUBERNETES_SERVICE_HOST']
 
 
+@fixture(scope='function')
+def set_github_ssh_private_key_env_var() -> None:
+    if SSH_PRIVATE_KEY_ENV_VAR not in os.environ:
+        private_key = Path.home() / '.ssh/id_rsa'
+        if private_key.exists():
+            os.environ[SSH_PRIVATE_KEY_ENV_VAR] = private_key.read_text()
+        else:
+            raise RuntimeError('cannot locate private SSH key to use for GitHub')
+
+
+@fixture(scope='function')
+def set_gitlab_ssh_private_key_env_var() -> None:
+    if SSH_PRIVATE_KEY_ENV_VAR not in os.environ:
+        private_key = Path.home() / '.ssh/id_rsa_e28827a593edd69f1a58cf07a7755107'
```

Comment:
Where does this funny name come from `id_rsa_e28827a593edd69f1a58cf07a7755107`?

Reply:
That is the filename that CircleCI gives the key (a hash of some sorts) because we now have multiple RSA keys. We have no control over this.

Reply:
Won't the tests fail locally then?